README.md

```diff
@@ -102,8 +102,9 @@ Please make sure you use Python 2.x. Python 3.x is not supported currently.
 To build for macOS, you need [Xcode 12 beta 3](https://developer.apple.com/xcode/resources/).
 The required version of Xcode changes frequently, and is often a beta release.
 Check this document or the host information on <https://ci.swift.org> for the
-current required version. Recently (as of 8-22-20), we have seen a bug that causes the build to fail
-if the Xcode app path includes any spaces. (See https://bugs.swift.org/browse/SR-13441 .)
+current required version.
+
+Swift's build tooling is meant to support spaces in the paths passed to them, but using spaces sometimes tickles bugs in Swift's build scripts or the tools they rely on. For example, https://bugs.swift.org/browse/SR-13441 is caused by a space in the Xcode path used on macOS. If you see Swift's build tooling misbehave due to a space in a path, please report the bug on the Swift bug tracker and then change the path to work around it.
```

Comment:
Nit: Here's a version that's better formatted.

```markdown
Swift's build tooling is meant to support spaces in the paths passed to them,
but using spaces sometimes tickles bugs in Swift's build scripts or the tools
they rely on. For example, [SR-13441](https://bugs.swift.org/browse/SR-13441)
is caused by a space in the Xcode path used on macOS. If you see Swift's build
tooling misbehave due to a space in a path, please
[report the bug on the Swift bug tracker](https://swift.org/contributing/#reporting-bugs)
and then change the path to work around it.

```

Reply:
(Compared to @airspeedswift's version above, mine also links "report the bug on the Swift bug tracker" to the bug reporting instructions on swift.org. I think that's an improvement; up to you whether you agree.)

Reply:
Thanks a lot, @brentdax!
Looks great to me.